IFC4x3/Quantities/w/Weight_2ueoe8QyrBduWuJHsi21zb/Documentation.md

```diff
@@ -1 +1 @@
-Total weight of reinforcing.
+Total weight of object
```

Comment:
Wondering how we have two Documentation files for weight.

Reply:
the other on is a phantom property andf will be deleted